package.json

```diff
@@ -20,6 +20,7 @@
     "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting"
   },
   "dependencies": {
+    "extend": "^3.0.0",
```

Comment:
I mean, I can write my own (or find it on the internets) but I wanted to see if you have something first. No big deal.